sopel/config/core_section.py

```diff
@@ -1136,6 +1136,32 @@ def homedir(self):
 
     """
 
+    timeout_ping_interval = ValidatedAttribute('timeout_ping_interval',
+                                               int,
+                                               default=50)
+    """The number of seconds before sending a PING command to the server.
+
+    :default: ``50``
```

Comment:
I think the option can be 0 or something. I need to test that, maybe I overdid something here.

Reply:
We have 2 options here:

* keep the default value to 50s, meaning that if timeout detection is modified, this need to be modified as well
* set a default value of 0, meaning that it will always use the /2.4 by default, and change the "default" in the docstring by "auto"

The second option looks nice, don't you think?

Reply:
> The second option looks nice, don't you think?

Yes, it does. The "magic" division by 2.4 makes the most sense unless someone is trying to work around a very finicky timeout implementation on some obscure ircd, in which case they can override that magic with this option.